packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -488,12 +492,823 @@ function Checkbox(props) {
+resize the column and double tapping to trigger a virtual click exits resizing.
+
+Note that we apply `visibility: hidden` to hide the resizer so that we reserve space for it in the column header at all times.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+import {useTableColumnResize} from '@react-aria/table';
+
+const Resizer = React.forwardRef((props, ref) => {
+  let {column, layoutState, onResizeStart, onResize, onResizeEnd, triggerRef, showResizer} = props;
+  let {resizerProps, inputProps} = useTableColumnResize({
+    column,
+    label: 'Resizer',
+    onResizeStart,
+    onResize,
+    onResizeEnd,
+    triggerRef
+  }, layoutState, ref);
+  // TODO: update the visual look of the resizer and have a focus style
+  return (
+    <div
+      role="presentation"
+      style={{
+        cursor: undefined,
+        width: '20px',
+        height: '20px',
+        border: '1px solid red',
+        touchAction: 'none',
+        flex: '0 0 auto',
+        boxSizing: 'border-box',
+        visibility: showResizer ? 'visible' : 'hidden'
+      }}
+      {...resizerProps}>
+      <VisuallyHidden>
+        <input
+          ref={ref}
+          type="range"
+          {...inputProps} />
+      </VisuallyHidden>
+    </div>
+  );
+});
+```
+
+</details>
+
+### MenuButton
+
+The `MenuButton` is used in the `ResizableTableColumnHeader` to display a list of available actions available to a column. It is built using the [useMenuTrigger](useMenu.html)
+and [useTreeState](/react-stately/useTreeState.html) hooks, and can be shared with many other components. Note that you'll have to call `continuePropagation()` in `onKeyDown`
+to properly bubble the `keydown` event up to `useSelectableCollection` for grid keyboard navigation.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+import {useMenuTriggerState} from '@react-stately/menu';
+import {useMenuTrigger} from '@react-aria/menu';
+import {Item} from '@react-stately/collections';
+
+// Reuse the Button, Menu, and Popover from your component library. See below for details.
+import {Button, Menu, Popover} from 'your-component-library';
+
+const MenuButton = React.forwardRef((props, ref) => {
+  // Create state based on the incoming props
+  let state = useMenuTriggerState(props);
+
+  // Get props for the button and menu elements
+  let {menuTriggerProps, menuProps} = useMenuTrigger({trigger: 'press'}, state, ref);
+  /*- begin highlight -*/
+  // Continue propagation of keydown event so that the left/right arrow key presses properly bubble to the table (useSelectableCollection)
+  let onKeyDown = (e) => e.continuePropagation();
+  /*- end highlight -*/
```

Comment:
another interesting bug (?) that I encountered. Our MenuTrigger avoids this problem since the `onKeyDown` from `useMenuTrigger` is sent via context and thus `useFocusable` won't automatically `stopPropagation` on it via `useKeyboard` https://github.com/adobe/react-spectrum/blob/8e57eaeda9112ea0783432409b8e6f9416d2fb28/packages/%40react-aria/focus/src/useFocusable.tsx#L74-L76

The aria implementation of a MenuButton runs into this problem if it doesn't do something similar. Maybe add `continuePropagation` to useMenuTrigger's onKeydown? Feels too specific though

Reply:
makes sense, that would conform with our philosophy of stopping propagation for events in general. Bit unfortunate that this is a "gotcha" for people implementing this in their tables. 

If we are amenable to easing up the propagation stuff, I could see an argument to continue propagation for keydowns we don't explicitly handle maybe?

Reply:
i think that would be fine, no reason to block keys we don't handle